index.html

```diff
@@ -0,0 +1,24 @@
+<!doctype html>
+
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <title>HTML & CSS Training</title>
+  <meta name="description" content="training">
+  <meta name="author" content="Wolox">
+  <link rel="stylesheet"  type="text/css" href="application.css">
```

Comment:
The application.css is not included in this PR, this is because your are ignoring it, as a result I get this layout when I run open the page in my laptop.
![image](https://user-images.githubusercontent.com/44579645/79488311-24cbec00-7fdf-11ea-9c97-8524cb1c452e.png)

Reply:
I ignored the `application.css` because this card is telling me that: https://trello.com/c/VDBPqcoy/11-setup-del-proyecto . I quote: 

> "Por ultimo como esto crea un par de cosas que no nos interesa que estén en el repositorio (como el application.css, la carpeta .sass-cache o el application.css.map entre otros) vamos a agregarlas al .gitignore para que no se suban."

I guess you should have to compile Sass on your machine to generate the actual `application.css` file, but it's just my opinion I'm not sure if that's like that.